format-translators-module/format-translators-module/test/test-structures/scg-visualisation-translator/a_lot_of_triples.scs

```diff
@@ -0,0 +1,1546 @@
+test_action_node
```

Comment:
This file is very big. You should not inflate the size of the codebase because of a single test. If you wanted to create a benchmark test, you need to create one using Google Benchmark. In general, such tests are mandatory for such components